pkg/scheduler/algorithm/predicates/metadata.go

```diff
@@ -39,33 +42,33 @@ type PredicateMetadata interface {
 	ShallowCopy() PredicateMetadata
 	AddPod(addedPod *v1.Pod, nodeInfo *schedulernodeinfo.NodeInfo) error
 	RemovePod(deletedPod *v1.Pod) error
+	GetInterPodPriorityNodeScore() map[string]int64
```

Comment:
I think that the idea behind this interface is that it stays independent from the algorithms. Is it possible to avoid this change? i.e., if you only access it inside a predicate, you can cast to the underlying type and use the struct's fields.